src/annotator/util/buckets.ts

```diff
@@ -89,7 +89,7 @@ export function computeAnchorPositions(anchors: Anchor[]): AnchorPosition[] {
 /**
  * Gap between the top/bottom of the container and the top/bottom of buckets.
  */
-export const BUCKET_BAR_VERTICAL_MARGIN = 30;
+export const BUCKET_BAR_VERTICAL_MARGIN = 25;
```

Comment:
With the new layout, 30 ended up being bigger than before. With 25, the space between the last button in the toolbar and the first bucket looks pretty much the same as before.